container/go/pkg/oci/test_index/blobs/sha256/a0cfcd4cc98a67def7ce9a0c7644d1c415d56d6d44c4a079a447f7eafb253048

```diff
@@ -0,0 +1 @@
+{"architecture": "amd64", "author": "Bazel", "config": {"Env": ["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin", "SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt"]}, "created": "1970-01-01T00:00:00Z", "history": [{"author": "Bazel", "created": "1970-01-01T00:00:00Z", "created_by": "bazel build ..."}, {"author": "Bazel", "created": "1970-01-01T00:00:00Z", "created_by": "bazel build ..."}], "os": "linux", "rootfs": {"diff_ids": ["sha256:01092e5921c5543a918d54d9df752ee09a84c912a1d914b7eb37e7152f20b951", "sha256:87c747af6dc3478b57493f1f3a2e0821696f8d56b86b1fef1128d1d74881cf7c"], "type": "layers"}}
```

Comment:
Are these image related files necessary? I don't see them used anywhere

Reply:
I added `test_index` as an example OCI index that `Read` can take in for testing purpose.

Reply:
I would add these files when you actually add the test instead of now.

Reply:
Ok